x-pack/solutions/security/plugins/security_solution/public/management/cypress/e2e/artifacts/host_isolation_exceptions_rbac.cy.ts

```diff
@@ -8,7 +8,7 @@
 import { getArtifactsListTestDataForArtifact } from '../../fixtures/artifacts_page';
 import { getArtifactMockedDataTests } from '../../support/artifacts_rbac_runner';
 
-describe(
+describe.skip(
```

Comment:
fyi, there should be no need to skip these tests, as the bugfix allowing this test to pass has been merged to main:
- https://github.com/elastic/kibana/pull/225678

(also, i'm reenabling these tests in 9.1 here: https://github.com/elastic/kibana/pull/228030)